shared/cfg/base.cfg

```diff
@@ -471,6 +471,10 @@ guest_dmesg_ignore = False
 # Whether to dump guest dmesg output to console
 guest_dmesg_dump_console = no
 
+# Guest dmesg white list that need to be ignored in case, multiple item can be seperated by ';'
+# For examples: hardware has not undergone testing by Red Hat and might not be certified;xx" 
+guest_dmesg_white_list = "this hardware has not undergone testing by Red Hat and might not be certified"
```

Comment:
let's pass either this as a list, so that we can iterate or atleast a comma(any other spl char) separated string and
split it while get it in `guest_dmesg_white_list = self.params.get("guest_dmesg_white_list").split(",")` something like this..otherwise this does work properly ,`for white_list_item in dmesg_white_list:`

Reply:
updated in the virt_vm.py, and get parameter as one list with spl char ";"